WordPress/src/main/res/layout/login_magic_link_request_screen.xml

```diff
@@ -0,0 +1,48 @@
+<RelativeLayout xmlns:android="http://schemas.android.com/apk/res/android"
```

Comment:
Just a quick tip about the layouts in general. If you select the whole layout and do `Code -> Reformat Code` in AS, it'll correct the alignments and order the xml properties. I personally like this a lot and try to do it each time I am done with a layout. You don't need to do it, just wanted to share :)